### ISSUE TITLE
Duplicated commands, but different behavior

### ISSUE THREAD

**Issue by johnrichardrinehart** [susam/emfy#4]
The README describes two ways to use `emacsclient`: a blocking and a non-blocking way. But, the command provided is the same for both. Probably there was a flag or an `&` missing for the non-blocking command. I'm on mobile (GitHub app) so it's hard to link to the line, unfortunately. But I copied some of the text:
```
 ​  ​-​ ​`emacsclient foo.txt bar.txt`​: Opens files in the existing Emacs
```

**Comment by egnha** [comment]
Indeed, you need to pass the optional argument `--no-wait` (`-n`).

**Comment by susam** [comment]
Fixed in commit f0a45e763e6a3c15c1bc3d12bebae3c90bff4df2. Thank you for reporting this issue.
